packages/sdk-codegen-scripts/src/miner.ts

```diff
@@ -74,13 +74,13 @@ export const filterAllFiles = (fileName: string) => fileName.trim().length > 0
  * @param fileName name of file from which to extract its extension
  */
 export const filterCodeFiles = (fileName: string) => {
-  if (/(^|\/)node_modules|lib\//gi.test(fileName)) return false
+  if (/(^|\/)node_modules\//gi.test(fileName)) return false
   const ext = path.extname(fileName).toLocaleLowerCase()
   // eslint-disable-next-line @typescript-eslint/no-use-before-define
   return ext in fileMiners
 }
 
-const skipFolder = (name: string) => /node_modules|lib/gi.test(name)
```

Comment:
Does this mean `lib` paths are now processed for example mining?

Reply:
My mistake. I just modified `skipFolder` to accept an exclude list